DevoxxClientMobile/src/main/java/com/devoxx/service/DevoxxService.java

```diff
@@ -646,8 +609,26 @@ private void retrieveExhibitionMapsInternal() {
     }
 
     @Override
-    public ReadOnlyListProperty<Sponsor> retrieveSponsors() {
-        return new ReadOnlyListWrapper<>(FXCollections.observableArrayList());
+    public ObservableList<Sponsor> retrieveSponsors() {
+        // TODO: pass the CfpEndpoint
```

Comment:
There are still TODOs here. Is still work that needs to be done and if so, what are they depending on?

Reply:
Currently the sponsors are fetched from a GCL RF which hits a rest endpoint. This endpoint is curated only for DEVOXX_FR.

Ideally, we would want this endpoint to accept the cfpEnpoint as a parameter and return the sponsors accordingly.